### PR TITLE
Ft welcome screen

### PR DESCRIPTION
**Pull request by bkandaris**
## Description
Added a form to add business information for the seller as well as a place to add a logo and brand color. 
## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/state/reducers/logoReducer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// import { UPLOAD_LOGO } from '../actionTypes';
```
Comment (ChiragThesia) [inline src/state/reducers/logoReducer.js:1]: Delete commented out code please!

**Review by ChiragThesia — APPROVED** [review]

**Comment by MatthewVaccaro** [comment]
Potential probs
Zipcode should only take numbers & a total of 5 characters. My assumption is Chirag will build the scheme to require those pieces and if a user adds in letters or more than 5 characters then it will error

Chore
The touch validation you have on fields turns on the second you start typing on any field.

**Comment by ChiragThesia** [comment]
@MatthewVaccaro I am just expecting a number. I will find a way to limit if need be, but for now there is no max.

**Review by ChiragThesia — DISMISSED** [review]
Looks good. We just need to discuss the back end aspect and how this will connect. We'll fix it this Friday!

**Comment by ChiragThesia** [comment]
@erferguson @bkandaris can you please check mark the read me up top. Thanks

**Comment by bkandaris** [comment]
Deleted unnecessary comments, fixed error handling on 'welcome', fixed redux issue with the logo.

**Review by Cireimu — CHANGES_REQUESTED** [review]
Checked out the welcome-screen and color-picker components.
There's a couple of things I want to be changed before we merge, besides these small things your guys' stuff looks pretty good. The main thing is I need you to pull from `develop` and resolve the merge conflicts, I can help guide you through which changes to accept.

Some notes:

- When I checked out your color picker component on its own route and the brand view route, I came across this warning message `Warning: Invalid DOM property 'classname'. Did you mean 'className'? ` Please fix this

- It also looks like your brand view route is a rough version of what it's supposed to be. Let's create a new task for that on the Trello, which can be started at any time but will need to be finished.

- Some sort of clear indicationon what's optional and what's required might be good to add, at the moment the fields that appear to be optional don't give to much of a clear signal that they are. It doesn't need to be done now, can be saved for a chore, but I thought it was worth noting nonetheless.

classname to className croped

**Review by Cireimu — APPROVED** [review]
Looks good, I have some notes for later, but looks good to go. Keep up the good work!

**Review by MatthewVaccaro — APPROVED** [review]
Other than the small visual bug/chore everything works
- The welcome screen has the correct inline valuation & when the submit button is click the information is store in redux correctly
- The brand page updates the visuals with the selected logo, the brand color picker works perectly and both store in redux correctly.

**Pull request merged**